modules/wallet.go

```diff
@@ -187,10 +187,9 @@ type (
 		Drop()
 	}
 
-	// Wallet stores and manages siacoins and siafunds. The wallet file is
-	// encrypted using a user-specified password. Common addresses are all
-	// dervied from a single address seed.
-	Wallet interface {
+	// EncryptionManager can encrypt, lock, unlock, and indicate the current
+	// status of the EncryptionManager.
+	EncryptionManager interface {
```

Comment:
the interface still has 25 functions, you just hid some of them.

the disadvantage is that it's confusing to define a separate interface when there isn't a clear use case for it. None of the other modules work with an `EncryptionManager` or a `KeyManager`; they all need the full `Wallet`. Maybe there's a stronger need for such interfaces inside the wallet package, but in that case, those interfaces should be defined there, instead of at the module level.